db/migrate/20240129231307_create_items.rb

```diff
@@ -0,0 +1,10 @@
+class CreateItems < ActiveRecord::Migration[6.1]
+  def change
+    create_table :items do |t|
+      t.string :description
+      t.integer :price
```

Comment:
when anything is monetary, you want to be a bit more specific than an integer. 
`:decimal, default: 0, precision: 13, scale: 2`
We know at Hnry that we are only servicing currencies that go to two decimal places, so we set a scale of 2. Precision is how many numbers are to the left of a decimal place. Common practice is 13.

Reply:
sweet, yep had just realised floats exist in Ruby so will add a new migration to update the field type to float. Thank you!

Reply:
I was planning to do a new migration to update the price field type to float but is decimal the correct data type?

Reply:
Yes decimal is preferred over float, specifically for the precision. A float is not an exact way to represent numbers and can lead to rounding errors.